app/models/task.rb

```diff
@@ -0,0 +1,11 @@
+class Task < ApplicationRecord
+  belongs_to :project
+  validates :project, presence: true
+  validates :title, presence: true
+  validate :deadline_after_creation
```

Comment:
👍

Reply:
But naming isn't good, it sounds like app should validate deadline after task creation